src/site/content/en/blog/compat2021-holiday-update/index.md

```diff
@@ -0,0 +1,150 @@
+[Interop 2022](https://github.com/web-platform-tests/interop-2022), which is the continuation of 
+this effort.
+{% endAside %}
+
+<div class="w-stats">
+ <div class="w-stat">
+   <p class="w-stat__figure">>90<sub>%</sub></p>
+   <p class="w-stat__desc">score in all browsers</p>
+ </div>
+</div>
+
+Since our [last update](https://web.dev/compat2021-midyear/), we’ve continued to see improvements 
+in all browsers. All browsers started with much lower test scores at the beginning of the year, but 
+now all browsers have surpassed 90%! This means the web platform has significantly improved 
+interoperability of 5 focus areas. 
+
+<figure class="w-figure">
+{% Img src="image/Wd2sVnt4VLho4jgp7UtIyWFceE02/Kaz3ye7gFfow8ia7lWYW.png", alt="A snapshot of Compat 
+2021 Dashboard (experimental browsers)", width="800", height="727" %}
+  <figcaption class="w-figcaption">
+    A snapshot of <a href="https://wpt.fyi/compat2021">Compat 2021 Dashboard</a> (experimental 
+    browsers).
+  </figcaption>
+</figure>
+
+{% Aside 'caution' %}
+The graph does not correctly show the improvements in Safari Technology Preview between May and 
+November, as the version tested was [stuck](https://github.com/web-platform-tests/wpt/issues/31147) 
+at an older version. This is now resolved but the data between May and November remains incorrect.
+{% endAside %}
+
+Contributions to browser engines are made not only by browser vendors, but also others in the web 
+community. For this project, we particularly want to thank Igalia for 
+[their involvement](https://www.igalia.com/2021/11/12/New-Interoperability-Milestones.html) and 
+their continued work to improve the scores. Igalia has contributed to improving all five focus 
+areas of Compat 2021.
+
+On [wpt.fyi](https://wpt.fyi/), the test results dashboard, there’s now a 
+[filtered test results view](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021)
+([permalink](https://wpt.fyi/results/?q=label%3Ainterop-2021&run_id=5629744973348864&run_id=5117500464300032&run_id=5734588178497536)) showing all of the tests included in Compat 2021, 
+and also views for [Chrome](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5738932147847168&run_id=5629744973348864), 
+[Firefox](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5644039006191616&run_id=5117500464300032), 
+and [Safari](https://wpt.fyi/results/?diff&filter=ADC&q=label%3Ainterop-2021&run_id=5739124314079232&run_id=5734588178497536) comparing the results to our last update in July.
+
+Let’s take a look at the improvements in each area!
+
+
+## CSS flexbox
+
+`flex-basis: content` is now on its way to all browsers, with implementations landing in 
+[Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=470421) and 
+[WebKit](https://trac.webkit.org/changeset/284440/webkit). (The `content` value was already 
+supported by Firefox.)
```

Comment:
```suggestion
supported by Gecko.)
```

Reply:
We should consistently use brand or repo names, and also help explain to those who don't know what the words mean.

Reply:
Does this not also imply that we should also be using Blink instead of Chromium?

Reply:
We used Chromium as the engine name in both https://web.dev/compat2021/ and https://web.dev/compat2021-midyear/.

Reply:
I should also say I think that's not a bug, Chromium is the name of a browser engine, and Blink is a part of Chromium. So when do we call it Blink instead of Chromium? I don't know :)